docs/questions/sharing/visualizing-results.md

```diff
@@ -27,87 +27,109 @@ To change the settings for a specific chart, for example a row chart, you could
+[Bar charts](./visualizations/line-bar-and-area-charts.md) are great for displaying a number grouped by a category (e.g., the number of users you have by country).
 
-## Trends
+![Bar chart](../images/bar.png)
 
-The [Trend](./visualizations/trend.md) visualization is great for displaying how a single number has changed between two time periods.
+## Combo charts
 
-![Trend settings](../images/trend-settings.png)
+[Combo charts](./visualizations/combo-chart.md) let you combine bars and lines (or areas) on the same chart.
+
+![Line + bar](../images/combo-chart.png)
 
 ## Detail
 
 The [Detail](./visualizations/detail.md) visualization shows a single result record (row) in an easy-to-read, two-column display.
 
 ![Detail of a record in the account table](../images/detail.png)
 
-## Progress bars
+## Funnel charts
 
-[Progress bars](./visualizations/progress-bar.md) are for comparing a single number to a goal value that you set.
+[Funnels](./visualizations/funnel.md) are commonly used in e-commerce or sales to visualize how many customers are present within each step of a checkout flow or sales cycle. At their most general, funnels show you values broken out by steps, and the percent decrease between each successive step.
 
-![Progress bar](../images/progress.png)
+![Funnel](../images/funnel.png)
 
 ## Gauges
 
 [Gauges](./visualizations/gauge.md) allow you to show a single number in the context of a set of colored ranges that you can specify.
 
 ![Gauge](../images/gauge.png)
 
-## Tables
+## Line charts
 
-The [Table](./visualizations/table.md) option is good for looking at tabular data (duh), or for lists of things like users or orders.
+[Line charts](./visualizations/line-bar-and-area-charts.md) are best for displaying the trend of a number over time, especially when you have lots of x-axis values. For more, check out our [Guide to line charts](https://www.metabase.com/learn/metabase-basics/querying-and-dashboards/visualization/line-charts) and [Time series analysis](https://www.metabase.com/learn/metabase-basics/querying-and-dashboards/time-series) tutorials.
 
-![Conditional formatting](../images/conditional-formatting.png)
+![Trend lines](../images/trend-lines.png)
+
+## Maps
+
+When you select the [Map](./visualizations/map.md) visualization, Metabase will automatically try and pick the best kind of map to use based on the table or result set.
+
+![Region map](../images/map.png)
+
+## Numbers
+
+The [Numbers](./visualizations/numbers.md) option is for displaying a single number, nice and big.
+
+![Number](../images/number.png)
+
+## Pie, donut, and sunburst charts
+
+A [pie chart or donut chart](./visualizations/pie-or-donut-chart.md) can be used when breaking out a metric by a single dimension, especially when the number of possible breakouts is small, like accounts by plan.
+
+A [sunburst chart](./visualizations/pie-or-donut-chart.md) chart is a pie chart with more than one ring to show the data broken out by additional dimensions.
```

Comment:
You're calling it "A sunburst chart chart is a pie chart", should remove the second chart.